09. cli-and-http-request/clihelper/WriterConfigurationStruct.go

```diff
@@ -0,0 +1,8 @@
+package clihelper
+
+type WriterConfiguration struct {
```

Comment:
Let's move all the structs to structs and this one to `structs\WriterConfiguration.go` @Nafrose

Reply:
Done